<?xml version="1.0" encoding="UTF-8"?>
<testng-results retried="3" ignored="0" total="4" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-08-11T12:26:34 IST" name="Default suite" finished-at="2022-08-11T12:27:02 IST" duration-ms="28086">
    <groups>
    </groups>
    <test started-at="2022-08-11T12:26:34 IST" name="Default test" finished-at="2022-08-11T12:27:02 IST" duration-ms="28086">
      <class name="com.crm.CreateOwner.CreateOwnerAndVerify">
        <test-method is-config="true" signature="dbConfig()[pri:0, instance:com.crm.CreateOwner.CreateOwnerAndVerify@791f145a]" started-at="2022-08-11T12:26:34 IST" name="dbConfig" finished-at="2022-08-11T12:26:34 IST" duration-ms="30" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- dbConfig -->
        <test-method is-config="true" signature="launchTheBrowser()[pri:0, instance:com.crm.CreateOwner.CreateOwnerAndVerify@791f145a]" started-at="2022-08-11T12:26:34 IST" name="launchTheBrowser" finished-at="2022-08-11T12:26:46 IST" duration-ms="12697" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchTheBrowser -->
        <test-method is-config="true" signature="loginToAppln()[pri:0, instance:com.crm.CreateOwner.CreateOwnerAndVerify@791f145a]" started-at="2022-08-11T12:26:47 IST" name="loginToAppln" finished-at="2022-08-11T12:26:48 IST" duration-ms="1234" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginToAppln -->
        <test-method retried="true" signature="VerifyTheOwnerCreatedRoomDetailsinTheRegisterdRoomList()[pri:0, instance:com.crm.CreateOwner.CreateOwnerAndVerify@791f145a]" started-at="2022-08-11T12:26:48 IST" name="VerifyTheOwnerCreatedRoomDetailsinTheRegisterdRoomList" finished-at="2022-08-11T12:26:52 IST" duration-ms="4139" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at com.crm.CreateOwner.CreateOwnerAndVerify.VerifyTheOwnerCreatedRoomDetailsinTheRegisterdRoomList(CreateOwnerAndVerify.java:52)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyTheOwnerCreatedRoomDetailsinTheRegisterdRoomList -->
        <test-method is-config="true" signature="logoutFromAppln()[pri:0, instance:com.crm.CreateOwner.CreateOwnerAndVerify@791f145a]" started-at="2022-08-11T12:26:52 IST" name="logoutFromAppln" finished-at="2022-08-11T12:26:52 IST" duration-ms="318" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoutFromAppln -->
        <test-method is-config="true" signature="loginToAppln()[pri:0, instance:com.crm.CreateOwner.CreateOwnerAndVerify@791f145a]" started-at="2022-08-11T12:26:52 IST" name="loginToAppln" finished-at="2022-08-11T12:26:53 IST" duration-ms="1042" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginToAppln -->
        <test-method retried="true" signature="VerifyTheOwnerCreatedRoomDetailsinTheRegisterdRoomList()[pri:0, instance:com.crm.CreateOwner.CreateOwnerAndVerify@791f145a]" started-at="2022-08-11T12:26:53 IST" name="VerifyTheOwnerCreatedRoomDetailsinTheRegisterdRoomList" finished-at="2022-08-11T12:26:55 IST" duration-ms="1633" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at com.crm.CreateOwner.CreateOwnerAndVerify.VerifyTheOwnerCreatedRoomDetailsinTheRegisterdRoomList(CreateOwnerAndVerify.java:52)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyTheOwnerCreatedRoomDetailsinTheRegisterdRoomList -->
        <test-method is-config="true" signature="logoutFromAppln()[pri:0, instance:com.crm.CreateOwner.CreateOwnerAndVerify@791f145a]" started-at="2022-08-11T12:26:55 IST" name="logoutFromAppln" finished-at="2022-08-11T12:26:55 IST" duration-ms="251" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoutFromAppln -->
        <test-method is-config="true" signature="loginToAppln()[pri:0, instance:com.crm.CreateOwner.CreateOwnerAndVerify@791f145a]" started-at="2022-08-11T12:26:55 IST" name="loginToAppln" finished-at="2022-08-11T12:26:56 IST" duration-ms="989" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginToAppln -->
        <test-method retried="true" signature="VerifyTheOwnerCreatedRoomDetailsinTheRegisterdRoomList()[pri:0, instance:com.crm.CreateOwner.CreateOwnerAndVerify@791f145a]" started-at="2022-08-11T12:26:56 IST" name="VerifyTheOwnerCreatedRoomDetailsinTheRegisterdRoomList" finished-at="2022-08-11T12:26:58 IST" duration-ms="1444" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at com.crm.CreateOwner.CreateOwnerAndVerify.VerifyTheOwnerCreatedRoomDetailsinTheRegisterdRoomList(CreateOwnerAndVerify.java:52)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyTheOwnerCreatedRoomDetailsinTheRegisterdRoomList -->
        <test-method is-config="true" signature="logoutFromAppln()[pri:0, instance:com.crm.CreateOwner.CreateOwnerAndVerify@791f145a]" started-at="2022-08-11T12:26:58 IST" name="logoutFromAppln" finished-at="2022-08-11T12:26:58 IST" duration-ms="463" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoutFromAppln -->
        <test-method is-config="true" signature="loginToAppln()[pri:0, instance:com.crm.CreateOwner.CreateOwnerAndVerify@791f145a]" started-at="2022-08-11T12:26:58 IST" name="loginToAppln" finished-at="2022-08-11T12:26:59 IST" duration-ms="950" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginToAppln -->
        <test-method signature="VerifyTheOwnerCreatedRoomDetailsinTheRegisterdRoomList()[pri:0, instance:com.crm.CreateOwner.CreateOwnerAndVerify@791f145a]" started-at="2022-08-11T12:26:59 IST" name="VerifyTheOwnerCreatedRoomDetailsinTheRegisterdRoomList" finished-at="2022-08-11T12:27:01 IST" duration-ms="1622" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at com.crm.CreateOwner.CreateOwnerAndVerify.VerifyTheOwnerCreatedRoomDetailsinTheRegisterdRoomList(CreateOwnerAndVerify.java:52)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyTheOwnerCreatedRoomDetailsinTheRegisterdRoomList -->
        <test-method is-config="true" signature="logoutFromAppln()[pri:0, instance:com.crm.CreateOwner.CreateOwnerAndVerify@791f145a]" started-at="2022-08-11T12:27:01 IST" name="logoutFromAppln" finished-at="2022-08-11T12:27:01 IST" duration-ms="314" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoutFromAppln -->
        <test-method is-config="true" signature="closeTheBrowser()[pri:0, instance:com.crm.CreateOwner.CreateOwnerAndVerify@791f145a]" started-at="2022-08-11T12:27:01 IST" name="closeTheBrowser" finished-at="2022-08-11T12:27:02 IST" duration-ms="642" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeTheBrowser -->
        <test-method is-config="true" signature="closeDBconfig()[pri:0, instance:com.crm.CreateOwner.CreateOwnerAndVerify@791f145a]" started-at="2022-08-11T12:27:02 IST" name="closeDBconfig" finished-at="2022-08-11T12:27:02 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeDBconfig -->
      </class> <!-- com.crm.CreateOwner.CreateOwnerAndVerify -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
